webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3587,7 +3584,89 @@ var userOverride = {
                         $applozic('.mck-dropup-menu').hide();
                 }
             };
-
+            _this.triggerWelcomeEvent = function(){
+                window.Applozic.ALApiService.sendMessage({
+                    data: {
+                        message: {
+                            type: 5,
+                            contentType: 10,
+                            message: 'Event: WELCOME',
+                            groupId: CURRENT_GROUP_DATA.tabId,
+                            metadata: {
+                                category: 'HIDDEN',
+                                KM_TRIGGER_EVENT: 'WELCOME',
+                            },
+                            source: 1,
+                        },
+                    },
+                    success: function (response) {
+                        console.log(response);
+                    },
+                    error: function (error) {
+                        console.error(error);
+                    },
+                });
```

Comment:
Please remove console.log

Reply:
console log is here for error handeling if API fails in any case.

Reply:
Removed